DESCRIPTION

```diff
@@ -80,6 +81,7 @@ Suggests:
     reldist,
     ade4,
     microbiomeDataSets,
+    miaTime,
```

Comment:
Is there reason for adding miaTime? This might cause a problem since packages are looked from Bioc or Cran. miaTime is only in Github so it cannot be installed here

(If miaTime is needed, there might be option for installing it with remotes package, but we have to check it)

Reply:
I guess this is because of the demo data sets that were needed. 

We can reconsider demo data sets from mia or miaViz if this helps..?

Reply:
Yes I added miaTime because the hitchip1006 data is loaded from that package. Are there better alternatives?

Reply:
I think there is Remotes also available in DESCRIPTION

Remotes:
Github:microbiome/miaTme

or something like that --> can you check

Reply:
Also it is better if you use data from mia package for instance. It is only demonstration for the functionality. We should avoid adding dependencies

Reply:
Ok! I did some extensive research for a good dataset, but the ones in mia were not ideal and did not give any significant results. I'll think a little bit.

Reply:
We could consider moving hitchip data to mia if it helps. It is not critical to have in miaTime.

Reply:
For now I could add miaTime to Remotes in DESCRIPTION. Or should we directly add hitchip to mia?

Reply:
What would be @TuomasBorman preference? I don't care too much which pkg hitchip data is in.

Reply:
You can add miaTime to Remotes for now. I have heard that mia might be problematic to install because of all the dependencies. In the future, we have to check which dependencies can be dropped, and also what we do with this